src/main/java/com/froobworld/farmcontrol/controller/entity/SnapshotEntity.java

```diff
@@ -19,19 +19,21 @@ public class SnapshotEntity {
     private final boolean customName;
     private final boolean tamed;
     private final boolean isPatrolLeader;
+    private final boolean isMob;
```

Comment:
Don't think we need this - we can check against the entity class if needed

Reply:
Done!